seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/catalog/AbstractJdbcCatalog.java

```diff
@@ -86,8 +86,11 @@ public abstract class AbstractJdbcCatalog implements Catalog {
 
     protected final Map<String, Connection> connectionMap;
 
+    protected final Boolean skipIndexWhenAutoCreateTable;
+
     public AbstractJdbcCatalog(
             String catalogName,
+            boolean skipIndexWhenAutoCreateTable,
```

Comment:
It's weird to put `skipIndexWhenAutoCreateTable` into constructor. It should be a parameter when invoke `createTable` method. Eg: 
```java
void createTable(TablePath tablePath, CatalogTable table, boolean ignoreIfExists, boolean createIndex)
```

Reply:
> It's weird to put `skipIndexWhenAutoCreateTable` into constructor. It should be a parameter when invoke `createTable` method. Eg:
> 
> ```java
> void createTable(TablePath tablePath, CatalogTable table, boolean ignoreIfExists, boolean createIndex)
> ```

Good advice, I will change it.